packages/ckeditor5-table/tests/tablecolumnresize/tablewidthresizecommand.js

```diff
@@ -71,11 +71,11 @@ describe( 'TableWidthResizeCommand', () => {
 		);
 	} );
 
-	it( 'should remove the attribute if new value was not passed', () => {
+	it( 'should remove the attributes if new value was not passed', () => {
```

Comment:
If `tableWidth` and `columnWidths` are not supposed to be connected anymore, I'd add tests for all combinations (missing `tableWidth`, missing `columnWidths`, both missing).